docs/getting_started/getting_started.rst

```diff
@@ -19,15 +19,15 @@ Install
    cd integration-test-framework
    pip install .
 
-`Pipenv <https://github.com/pypa/pipenv>`__ is more appropriate for
+`Poetry <https://python-poetry.org/>`__ is more appropriate for
 developers as it automatically creates virtual environments.
 
 .. code:: bash
 
    git clone https://dev-bosch.com/bitbucket/scm/pea/integration-test-framework.git
    cd integration-test-framework
-   pipenv install --dev
-   pipenv shell
+   poetry install
```

Comment:
we are in thee user installation s maybe the dev dependencies are not needed and user has to write : 
poetry install --no-dev

Reply:
So then maybe delete this part from the getting_started because we have a page dedicated to devs.

Reply:
Even though I agree with you, it's still weird to skip the dev dependencies as right after we show the tasks to invoke, and invocations is part of the dev dependencies...

Reply:
But you made me notice that we still had old bitbucket links in there, thanks! :D

Reply:
I replaced the entire dev part with a simple pykiso example with options